meetings/2023-10-11.md

```diff
@@ -44,17 +57,62 @@
+* RG: I wanted to ask if those remarks are public or were submitted directly to the W3C team.
+* PAC: Members can choose to make the responses public but are not required. This is a synthesis of public and non-public responses.
+* SC: For some of the significant feedback requiring to update charter, clarify process, etc. 
+* SC: We need to update editors draft to be updated with current charter. If you see that some specific specs are not doing what they are supposed to do please let us know.
+* SC: Charter is referring to the TRs, to updated them we need to make a new release. WG can look at either editor drafts or latest publish versions on solidproject.org/TR
+* SC: I see most of the points as straight forward to address. Including broadening the scope. If you could come up with concrete issues we could propose PRs addressing those specific issues. We don't want to randomly propose things.
+* PAC: My goal is to create an issue for each one of those items. Whenever I can include more details I will do it. For public votes I will copy and paste all the details.
+* PAC: We probably want to use `solid/solid-wg-charter` for those PRs no to create to much activity on W3C charters repo.
+* PAC: I'm not worried about the contents of the cg.
+* eP: Regarding dependencies. Our latest approach to was to mark them as optional deliverables.
+* PAC: We don't have a clear list of deliverables. Depending on the maturity of the specs is transferring the authority to the CG. Here the criticism on different process between CG and WG is valid. We should think which specifications WG can bring to REC. We should decide for the rest if we make them soft or optional dependencies. 
+* eP: Extending the timeline, would it be feasible that the dependencies are necessary for Solid. Instead of depending on CG or whatever else.
+* SC: https://github.com/solid/solid-wg-charter/issues/37#issuecomment-1566833007
+* SC: This is precisely the thing that we needed a feedback from W3C. If either the charter is accepted with all of those deliverables, or there needs to be understanding that Solid Protocol still can get to REC irrespective to the status of the dependencies.
+* SC: The current list of deliverables is already a lot for a WG. I took ages to get to the point where the specs are now. For example for Solid Protocol there has been few months of waiting for the next release since there are not reviews being submitted. What guarantee would WG participants can give that they'll actually get things done when they didn't in the CG.
+* SC: W3C needs to tell us what is acceptable. I linked to some examples, AS2 or ActivityPub which link to some dependencies which were not mature enough at that time.
```

Comment:
```suggestion
  * ... This is precisely the sort of thing where we need feedback from W3C — whether the charter is accepted with all of those deliverables, or there needs to be some understanding that Solid Protocol still can get to REC irrespective of the status of the dependencies.
  * ... The current list of deliverables is already a lot for a WG. I took ages to get to the point where the specs are now. For example for Solid Protocol there has been few months of waiting for the next release since there are not reviews being submitted. What guarantee would WG participants can give that they'll actually get things done when they didn't in the CG.
  * ... W3C needs to tell us what is acceptable. I linked to some examples, AS2 or ActivityPub which link to some dependencies which were not mature enough at that time.
```